### ISSUE TITLE
KeyError: 'list_stream_input_device' of process_online function (madmom0.17.dev0)

### ISSUE THREAD

**Issue by snownstone** [CPJKU/madmom#471]
### Expected behaviour

Run  [beat tracking code by Ian Charnas](https://groups.google.com/g/madmom-users/c/yYtp6Y43yWQ). (Thank Ian.)

### Actual behaviour

KeyError as shown below.
![Selection_012](https://user-images.githubusercontent.com/61407739/112127631-9a353900-8c00-11eb-8022-fa531756b2aa.png)

I think this issue is related to this [PR](https://github.com/CPJKU/madmom/pull/451/commits/bc644fc2a7638324f8e96c057b394612169010af). I can use list_stream_input_device perfectly in terminal with script in _bin_. But I did not figure out why this error happened here. I solved it by turn the related codes to comments in the processors.py and reinstall madmom from the source code.

Madmom is really awesome. Thank you! 

### Steps needed to reproduce the behaviour

Install madmom==0.17.dev0 from the source and run the code.

**Comment by superbock** [comment]
Yes, this is a bug, I'll fix it in the next release (which btw should be only within months ;))
